Map/changelog.txt

```diff
@@ -1,3 +1,4 @@
 changelog = {
   [[Initial crowdmap revision]]
```

Comment:
Need a sneaky little comma now that there's two items in the changelog -

```suggestion
  [[Initial crowdmap revision]],
```

Reply:
Done